modules/media.js

```diff
@@ -206,11 +252,11 @@ Media.prototype.get = function(type, directory, forEach) {
   }
 
   if (typeof(directory) !== 'string') {
-    if (window.ffosbr.utils.isFunction(directory)) {
+    if (ffosbr.utils.isFunction(directory)) {
       // Parameter "directory" was not provided, and the
       // second parameter is really the "forEach" function
       forEach = directory;
-    } else if (window.ffosbr.utils.isFunction(forEach)) {
+    } else if (ffosbr.utils.isFunction(forEach)) {
       throw new Error('Missing or invalid directory');
     } else {
       throw new Error('Missing or invalid callback');
```

Comment:
Both are needed.
In the first case, it's confirming that although the directory parameter is a function, if forEach is also a function we assume that the forEach parameter is the true forEach function. Otherwise, we can assume the directory parameter was left out, and the second parameter is actually the forEach function.
In the second case, the directory parameter was valid and we just need to confirm that the forEach parameter is also valid.

If you can think of a more efficient way of making these checks (maintaining optional parameters), let me know.